CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [1.4.1]
+* Update Native Auth logging levels for improved clarity and consistency. #2184
```

Comment:
Do you know what are the guidelines about having the reference to the PR in the changelog?

It would be nice to either remove them or keep them for all of the changes (to improve consistency).